scripts/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+export NODE_OPTIONS=--openssl-legacy-provider
+cd ..
+rm -Rf dist
+rm -f ~/.cncrc
+rm -fR ~/cncjs_i18n_old
+mkdir -p ~/cncjs_i18n_old/{app,server}
+
+cp -R src/app/i18n ~/cncjs_i18n_old/app
+cp -R src/server/i18n ~/cncjs_i18n_old/server
+
+yarn install
+yarn build-prod
+cd dist/cncjs/
```

Comment:
There's a neat trick you can use here:
```sh
pushd dist/cncjs
...
popd
```

This will move between directories then restore your original position without having to `cd ..` twice (or more)!

Reply:
I agree it will be more beautiful, corrected [6e514d2](https://github.com/cncjs/cncjs/pull/837/commits/6e514d28f6bf216a1f87c607aece9c6ccf0c64b4)